evm-contracts/src/v0.7/dev/StalenessFlaggingValidator.sol

```diff
@@ -76,28 +76,11 @@ contract StalenessFlaggingValidator is ConfirmedOwner {
    * @notice Check for staleness in an array of aggregators
    * @dev If any of the aggregators are stale, this function will return true,
    * otherwise false
-   * @param aggregators address[] memory
+   * @param aggregators address[] calldata
    * @return address[] memory stale aggregators
    */
-  function check(address[] memory aggregators) public view returns (address[] memory) {
-    uint256 currentTimestamp = block.timestamp;
-
-    address[] memory staleAggregators = new address[](aggregators.length);
-    uint256 staleCount = 0;
-    for (uint256 i = 0; i < aggregators.length; i++) {
-      address aggregator = aggregators[i];
-      if (isStale(aggregator, currentTimestamp)) {
-        staleAggregators[staleCount] = aggregator;
-        staleCount++;
-      }
-    }
-
-    if (aggregators.length != staleCount) {
-      assembly {
-        mstore(staleAggregators, staleCount)
-      }
-    }
-    return staleAggregators;
+  function check(address[] calldata aggregators) external view returns (address[] memory) {
```

Comment:
`check` and `update` are essentially type-explicit versions of the new Keeper functions. There's an open question as to whether we should remove them or not.